python/name-service/pyproject.toml

```diff
@@ -7,13 +7,13 @@ authors = ["Honeycomb"]
 [tool.poetry.dependencies]
 python = "^3.10"
 flask = "^2.3.2"
-python-dotenv = "^1.0.0"
+python-dotenv = "^1.0.1"
 requests = "^2.31.0"
-opentelemetry-api = "^1.16.0"
-honeycomb-opentelemetry = "0.1.2b0"
-opentelemetry-instrumentation-requests = "0.37b0"
-opentelemetry-instrumentation-flask = "0.37b0"
+opentelemetry-api = "^1.22.0"
+honeycomb-opentelemetry = "0.4.0b0"
```

Comment:
latest honeycomb distro is only compatible with 1.22.0/0.43b0, so those otel dependencies are different than the other updates